index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile first Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./image/static/Inter-Black.ttf"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <main class="main1">
+        <section class="work">
+         <div class="container">
+         <div class="work1"><h1>My Recent Works</h1></div>
+         <div class="vector"><img src="./image/Vector.png" alt=""></div>
+         <div class="Img"><img src="./image/Img Placeholder.png" alt=""></div>
+         <div class="Multi-Post"><h1>Multi-Post Stories</h1></div>
+         <div class="para"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
+         <div class="container1">
+            <div class="item1">css</div>
+            <div class="item2">html</div>
+            <div class="item3">bootstrap</div>  
+            <div class="item2">Ruby</div>  
+          </div>
```

Comment:
- [ ] Kindly use `<ul>` and `<li>` tags to group and list your technologies. This is important for semantic reasons